### PR TITLE
create: updated to oauth

### PR DESCRIPTION
**Pull request by bnert**

### CODE REVIEW

File: src/jama/create/create.py [hunk 1]
```diff
@@ -46,10 +49,17 @@ def from_dialog(url_base, json_to_parse):
         to_post_obj["itemType"] = int(sub_data["itemType"])
         to_post_obj["fields"]["name"] = sub_data["newItemName"]
         to_post_obj["fields"]["description"] = sub_data["description"]
-        to_post_obj["fields"]["assignedTo"] = sub_data["asignee"]
 
-        jama_resp = requests.post(jama_url, json=to_post_obj, auth=(os.environ['JAMA_USER'], os.environ['JAMA_PASS']))
-            
+        if "DB_HOST" not in os.environ:
```
Comment (yejingyu) [inline src/jama/create/create.py]: You can simply use the `api_caller.post()` in `jama/api_caller` to do the same thing.
I think using the same caller function can make our code more maintainable.

File: src/jama/create/create.py [hunk 2]
```diff
@@ -143,8 +153,15 @@ def from_text(base_url, content_dict):
         
 
     url = base_url + '/rest/latest/items/'
-    jama_resp = requests.post(
-        url, json=to_post_obj, auth=(os.environ['JAMA_USER'], os.environ['JAMA_PASS']))
+    if "DB_HOST" not in os.environ:
```
Comment (yejingyu) [inline src/jama/create/create.py]: Similar to line 53.
You can simply use the `api_caller.post()` in `Jama/api_caller` to do the same thing.
I think using the same caller function can make our code more maintainable.

**Review by yejingyu — DISMISSED** [review]
Is there any reason of why we don't use the api_caller.post()?

File: src/jama/create/create.py (hunk 2 above)
Comment (bnert) [reply src/jama/create/create.py]: @yejingyu done

**Pull request merged**